content/03.methods.md

```diff
@@ -4,66 +4,67 @@
+Unlike Pubtator Central, these filtered preprints do not contain any annotations.
+Therefore, we used TaggerOne [@doi:10.1093/bioinformatics/btw343] to tag every chemical and disease entity and GNormplus [@doi:10.1155/2015/918710] to tag every gene and species entity for our preprint set.
+Once tagged, we used spacy to preprocess every preprint in the same medium as described in our Pubtator Central section.
 
 ## Constructing Word Embeddings for Semantic Change Detection
 
 Word2vec [@arxiv:1301.3781] is a natural language processing model designed to model words based on their respective neighbors in the form of dense vectors.
 This suite of models comes in two forms, a skipgram model and a continuous bags of words (CBOW) model.
 The skipgram model generates these vectors by having a shallow neural network predict a word's neighbors given the word, while the CBOW model predicts the word given its neighbors.
-We used the CBOW model to construct these word vectors for this project; however, the skipgram model can also be used.
-These word2vec models have stability issues within year and across years [@arxiv:1804.09692; @doi:10.1007/978-3-030-03991-2_73; @doi:10.1162/tacl_a_00008; @doi:10.18653/v1/S18-2019].
-We took an intra-year and inter-year approach to correct for this instability.
-For each year, we trained ten different CBOW models using the following parameters: vector size of 300, 10 epochs, minimum frequency cutoff of 5, and a window size of 16 for abstracts.
-Every model has its own unique embedding space following training, making it difficult to compare two models without a correction step.
+We used the CBOW model to construct word vectors for each year.
+Despite the power of these word2vec models, it has been shown that they have stability issues within year and across years [@arxiv:1804.09692; @doi:10.1007/978-3-030-03991-2_73; @doi:10.1162/tacl_a_00008; @doi:10.18653/v1/S18-2019].
+To correct this instability, we took an intra-year and inter-year approach.
+Each year, we trained ten different CBOW models using the following parameters: vector size of 300, 10 epochs, minimum frequency cutoff of 5, and a window size of 16 for abstracts.
+Every model has its own unique vector space following training, making it difficult to compare two models without a correction step.
 We used an alignment technique termed Orthogonal Procrustes [@doi:10.1007/BF02289451] to account for this issue.
-We aligned all trained word2vec models for both datasets to the first model trained in the year 2021.
-After alignment, we used each corrected model to detect changes in a word's neighboring context.
+We aligned all trained CBOW models for the Pubtator Central dataset to the first model trained in 2021.
+Likewise, we aligned all CBOW models for the BioRxiv/MedRxiv dataset to the first model trained in 2021.
+We used Parametric UMAP [@arxiv:2009.12981] to visually confirm our alignment approach made sense.
```

Comment:
>I don't fully understand this - don't you have to do regular umap first to get the info to train parametric umap?

The difference here is the way embeddings are constructed. Both model generates a graph based on the datapoint distances. Following the graph construction, regular UMAP generates embeddings directly from the graph while Parametric UMAP trains a neural network on the generated graph to build embeddings.

Reply:
In that case, should we say that you used UMAP to inspect the alignment approach? Since what really matters is the UMAP step - maybe I'm confused. I don't see any barriers to continuing on with writing and circling back to this as a discussion point at our next meeting.